### PR TITLE
Admin branch

### PR DESCRIPTION
**Pull request by anhhuynhUNC**
pull request #2, added a bunch of stuff

### CODE REVIEW

**Comment by ShivramR** [comment]
This is great work! Is there a way you can open an alert() confirmation on delete? I accidentally deleted a post I'd input there but it looks really good

File: admin-app/src/Insert/CagetorySelect.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+export default function CagetorySelect(props) {
+    
+
+    return (
+        <div className="insert-select">
+            <select onChange={(e) => {props.callback(e)}}>
+                <option value="DEFAULT" disabled>Choose cagetory</option>
+                <option value="Workplace Updates">Workplace Updates</option>
+                <option value="Policy Links">Policy Links</option>
+                <option value="Instruction Videos">Instruction Videos</option>
```
Comment (ShivramR) [inline admin-app/src/Insert/CagetorySelect.js]: Also can these be changed to "Instructional" videos?

File: admin-app/src/Utilities/PanelColor.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        case undefined:
+            color = "#00594C";
+            break;
+        case "Workplace Updates":
+            color = "#00594C";
+            break;
+        case "Policy Links":
+            color = "#EF426F";
+            break;
+        case "Instruction Videos":
```
Comment (ShivramR) [inline admin-app/src/Utilities/PanelColor.js]: And here as well - Instructional

**Comment by gda975** [comment]
Could you also give the admin the option to add their own personal title to a post in addition to the hard-coded title indicating what kind of post it is. Also, it would be good to give the admin a checkbox on each post to indicate whether they want to notify users about the new post.

**Comment by ShivramR** [comment]
Also another thing I noticed was that the word is supposed to be "category"/"categories" and there seem to be typos with that throughout the project directory, could you fix that really quick?

**Pull request closed without merging**